fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1113,10 +1113,8 @@ private void startNonLeaderDaemonThreads() {
             // metastoreEventsProcessor.init();
             // metastoreEventsProcessor.start();
         }
-        if (!usingNewPrivilege) {
-            // domain resolver
-            domainResolver.start();
-        }
+        // domain resolver
+        domainResolver.start();
```

Comment:
domainResolver only inited when usingNewPrivilege is true. should check nullable?

Reply:
domainResolver will initialize no matter whether usingNewPrivilege is true or false. I have two separate constructors for each case.